src/Sentry/Internal/Hub.cs

```diff
@@ -531,13 +531,13 @@ public void CaptureCodeLocations(CodeLocations codeLocations)
     }
 
     /// <inheritdoc cref="IMetricHub.StartSpan"/>
-    public ISpan StartSpan(string operation, string description)
+    public ISpan StartSpan(string name, string operation, string description)
```

Comment:
With this, we're required to provide a name for every span. And that name gets discarded if there's already a transaction running.

Reply:
Kind of... there's an extension method that just takes two parameters (operation and description). But you wouldn't know whether to call that one or the other one unless you knew whether you were creating a Transaction or a Span. The whole point of the StartSpan method is so that you don't have to make that distinction. 

In the case of Transactions/Spans for HttpClient requests, the Transaction.Name will be the same as the Transaction.Description, but that won't be the case for other transactions... so it was necessary to add a third "name" parameter (or it will be, as soon as this method is used beyond HttpClient spans).